inst/scripts/r_remote_eval.R

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env Rscript
+library(civis)
+
+
+args <- commandArgs(trailingOnly = TRUE)
+task_file_id <- as.numeric(args[[1]])
+
+# suppresses a warning about namespace future not being available,
+# and being masked by .Globalenv.
+# nothing from future is used here, so I'm suppressing this warning rather than installing/requiring future.
+suppressWarnings(fut <- read_civis(task_file_id))
+
+# install and load missing packages
+cat("Installing and loading required packages", fill = TRUE)
+
+pkgs <- fut$packages
+if (length(pkgs) > 0) {
+  pkg_list <- pkgs[!(pkgs %in% rownames(installed.packages()))]
+  if (length(pkg_list) > 0) install.packages(pkg_list, repos = "https://cloud.r-project.org")
+  req_results <- lapply(fut$packages, require, character.only = TRUE, quietly = TRUE)
```

Comment:
What is the reason to use `require` and `quietly` here?  Are we safe from package installation errors?

Reply:
This just suppresses any package loading messages. Any installation errors would still pop up.